releases/zeppelin-release-0.6.0.md

```diff
@@ -0,0 +1,105 @@
+---
+layout: page
+title: "Apache Zeppelin Release 0.6.0"
+description: ""
+group: release
+---
+<!--
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+{% include JB/setup %}
+
+## Apache Zeppelin Release 0.6.0
+
+The Apache Zeppelin community is pleased to announce the availability of the 0.6.0 release.
+
+The community put significant effort into improving Apache Zeppelin since the last release, focusing on having new backend support, implementing authentication and authorization for enterprise.
+More than 70+ contributors provided 360+ patches for new features, improvements and bug fixes.
+More than 200+ issues have been resolved.
+
+We encourage to [download](../../download.html) the latest release. Feedback through the [mailing lists](../../community.html) is very welcome.
+
+### Backend support
+
+This release includes new backend support for
+
+   * [Alluxio](../../docs/0.6.0/interpreter/alluxio.html)
+   * [Hbase](../../docs/0.6.0/interpreter/hbase.html)
+   * [HDFS](../../docs/0.6.0/interpreter/hdfs.html)
+   * [JDBC](../../docs/0.6.0/interpreter/jdbc.html)
+   * [Livy](../../docs/0.6.0/interpreter/livy.html)
+   * [Python](../../docs/0.6.0/interpreter/python.html)
+   * [SparkR](../../docs/0.6.0/interpreter/r.html)
```

Comment:
The reason I wrote as `SparkR` here is that R interpreter belongs to spark group. I believe Zeppelin will have R interpreter under r group one day, like the way that general python interpreter is added recently apart from pyspark interpreter. What do you think?